ark/segmentation/marker_quantification.py

```diff
@@ -43,6 +54,10 @@ def compute_marker_counts(input_images, segmentation_labels, nuclear_counts=Fals
     if 'label' not in regionprops_features:
         regionprops_features.append('label')
 
+    # centroid is required
+    if 'centroid-0' not in regionprops_features and 'centroid' not in regionprops_features:
```

Comment:
Why are we checking for centroid-0 and centroid? Centroid-0 is the output for the first coord, centroid is the input name

Reply:
I think past Adam assumed centroid-0 and centroid-1 could get passed as `regionprops_features`, noticed that just a check for 'centroid' would miss that case and consequently create two copies of 'centroid-0' and 'centroid-1' later on, so he lazily compensated with a check for 'centroid-0'.

Basically a hacky 'fix' to a rare problem.  Less hacky fix would be to check for 'centroid' in any element of regionprops_features